app/views/layouts/application.html.slim

```diff
@@ -10,6 +10,8 @@ html
 
     = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    - if controller_with_specific_js?
+      = javascript_include_tag "controller/#{controller_name}"
```

Comment:
Hmm I don't think it's worth it to have separate JS files per controller for this kind of app, especially because at the moment there's no controller-specific JS code and you're loading the same libraries (=duplicating them). What was your motivation for doing this?

Reply:
Actually not, I only load an extra js in case there is one with the controller name in the controller directory. And there is js only used on some controllers pages. In fact all self written js, not from bootstrap is only for specific pages.

Reply:
I reverted that. So its out of the PR.